src/store/modules/app.ts

```diff
@@ -153,14 +162,31 @@ const actions = {
   getProposal: async ({ commit }, { space, id }) => {
```

Comment:
This method and the getResults method don't have side effect, (no state change) could you move them in a helper file like helpers/snapshot.ts? This will make easier to reuse.